packages/frontend/src/components/message/Message.tsx

```diff
@@ -1191,13 +1191,15 @@ function WebxdcMessageContent({
         // because there is a button below that does the same
       />
       <div
-        className='name'
+        className='info-text'
         title={`${info.document ? info.document + ' \n' : ''}${info.name}`}
       >
-        {info.document && truncateText(info.document, 24) + ' - '}
-        {truncateText(info.name, 42)}
+        <div className='document'>
+          {info.document && truncateText(info.document, 32)}
```

Comment:
`truncateText` seems to be unnecessary now, right? I'm not suggesting to remove it, but just to understand what's going on. If so, maybe it's worth adding a comment?